app/models/guide.rb

```diff
@@ -37,28 +39,114 @@ def search_data
     as_json only: [:name, :overview, :crop_id]
   end
 
+  def basic_needs
+    if !user.gardens.empty?
+      # We should probably store these in the DB
+      basic_needs = [{
+          name: 'Sun / Shade',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.